zookeeper-server/src/main/java/org/apache/zookeeper/server/EphemeralType.java

```diff
@@ -212,7 +212,8 @@ public static void validateServerId(long serverId) {
      */
     public static void validateTTL(CreateMode mode, long ttl) {
         if (mode.isTTL()) {
-            TTL.toEphemeralOwner(ttl);
+            // nothing to do in 3.5 was:
+            // TTL.toEphemeralOwner(ttl);
```

Comment:
We could remove the if-else, but than we would have to be extra careful with backporting. So, works for me!

Reply:
@eolivelli How is this code change related to this patch?